index.js

```diff
@@ -94,6 +97,35 @@ export class DbApi {
       forks: 'forks' in doc && doc.forks.length ? doc.forks.join(',') : null,
     }
     this.#writeDocSql.run(flattenedDoc)
+
+    if (this.#listeners.has(doc.version)) {
+      process.nextTick(() => {
+        const set = this.#listeners.get(doc.version)
+        if (set) {
+          for (const listener of set.values()) {
+            listener(doc)
+          }
+          this.#listeners.delete(doc.version)
+        }
+      })
+    }
```

Comment:
This is just an abundance of caution (perhaps unnecessary):

Avoid failure if `doc` is mutated (e.g. version is changed) before the next tick by referencing the version directly (I know it's highly unlikely that we would do this, but if the code ended up doing this, then `doc.version` would refer to the mutated `version` in the next tick)

```suggestion
    const { version } = doc
    if (this.#listeners.has(version)) {
      process.nextTick(() => {
        const set = this.#listeners.get(version)
        if (set) {
          for (const listener of set.values()) {
            listener(doc)
          }
          this.#listeners.delete(version)
        }
      })
    }
```